### PR TITLE
Fix bug 1447772: Require fraud report attachment to be images

### PR DESCRIPTION
**Pull request by pmac**
Also move legal tests so that py.test will find them

### CODE REVIEW

File: bedrock/legal/forms.py [hunk 1]
```diff
@@ -97,9 +97,7 @@ class FraudReportForm(forms.Form):
             }
         )
     )
-    input_attachment = forms.FileField(
-        required=False,
-    )
+    input_attachment = forms.ImageField(required=False)
```
Comment (jpetto) [inline bedrock/legal/forms.py]: From the docs, it appears this auto-validates format to ensure the file is of a type understood by Pillow. Is that right?

Comment (pmac) [reply bedrock/legal/forms.py]: That is my understanding as well. And there is a test that tries to upload a text file and fails.

**Comment by pmac** [comment]
@jpetto think I should change the display name on the form from "Attachment" to e.g. "Image Attachment"? There are probably l10n issues with that though...

File: bedrock/legal/forms.py (hunk 1 above)
Comment (jpetto) [reply bedrock/legal/forms.py]: I guess my question is: will this allow things like .pdf and .tiff? Do we want to allow those kinds of images? Should we force a restriction to the three "standards": .png, .gif, and .jpg?

**Comment by pmac** [comment]
The error message when you upload a non-image does say "Upload a valid image. The file you uploaded was either not an image or a corrupted image." UX isn't the best, but it's not bad either. Probably fine.

**Comment by jpetto** [comment]
> think I should change the display name on the form from "Attachment" to e.g. "Image Attachment"? There are probably l10n issues with that though...

I do think this is a good idea, but a bummer for L10n. Could be put behind a tag though?

**Comment by pmac** [comment]
Oh. It seems that page is localizable, but not localized. So I can probably change the string.

**Comment by pmac** [comment]
Okay. I updated the help note to prepend `Images only.`

File: bedrock/legal/forms.py (hunk 1 above)
Comment (pmac) [reply bedrock/legal/forms.py]: Might allow .tiff, but doesn't allow .pdf.

Comment (pmac) [reply bedrock/legal/forms.py]: These are the supported formats http://pillow.readthedocs.io/en/latest/handbook/image-file-formats.html

Comment (pmac) [reply bedrock/legal/forms.py]: I believe we could filter on image type if we need to.

Comment (jpetto) [reply bedrock/legal/forms.py]: That's quite the list! I think we're pretty well covered. We can limit at a later date if needed.

**Comment by glogiotatidis** [comment]
lgtm, nice work!

**Pull request merged**